com.unity.uiwidgets/Runtime/ui/window.cs

```diff
@@ -257,8 +257,10 @@ public class Window {
             return null;
         }
 
-        List<string> _computePlatformResolvedLocale(List<string> supportedLocalesData) =>
-            Window_computePlatformResolvedLocale(supportedLocalesData);
+        // Window_computePlatformResolvedLocale not implement yet, TODO _computePlatformResolvedLocale
+        List<string> _computePlatformResolvedLocale(List<string> supportedLocalesData) {
+            return Window_computePlatformResolvedLocale(supportedLocalesData)?? new List<string>();
```

Comment:
we should also change "static extern List<string> Window_computePlatformResolvedLocale(List<string> supportedLocalesData);" =>  "static extern void Window_computePlatformResolvedLocale(List<string> supportedLocalesData);"

The key issue is, we should keep the consistence between the C# and C++ parts of a PInvoke